.github/workflows/webdriver-test.yml

```diff
@@ -4,18 +4,27 @@ on: [push, pull_request]
 
 jobs:
     build:
-      runs-on: ubuntu-latest
+      runs-on: ubuntu-24.04-arm
```

Comment:
ubuntu-latest works for amd64, which has some issues with interacting with webdriver io that is on arm.